src/api/resources/validate.ts

```diff
@@ -73,8 +73,8 @@ const validateWaypoint = (r: Waypoint): boolean => {
   if (typeof r.position === 'undefined') {
     return false
   }
-  if (!isValidCoordinate(r.position)) {
-    return false
+  if (isValidCoordinate(r.position)) {
+    return true
```

Comment:
After futher review.... changing this test to return true here will not then test the `feature` if it is present.
Was there a reason for exiting the test here?

Reply:
Ignore this... I'm not sure why I thought that this was an issue....